browser/ui/webui/brave_education/brave_education.mojom

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// based on //chrome/browser/ui/webui/whats_new/whats_new.mojom
+
+module brave_education.mojom;
+
+import "url/mojom/url.mojom";
+import "mojo/public/mojom/base/time.mojom";
+
+// Used by the WebUI page to bootstrap bidirectional communication.
+interface PageHandlerFactory {
+  // The WebUI page's |BrowserProxy| singleton calls this method when the
+  // page is first initialized.
+  CreatePageHandler(pending_remote<Page> page,
+                    pending_receiver<PageHandler> handler);
+};
+
+// Browser-side handler for requests from WebUI page.
+interface PageHandler {
+  // Returns the URL of the What's New page to show.
+  GetServerUrl() => (url.mojom.Url url);
+
+  // Record metric when browser commands are executed.
+  RecordBrowserCommandExecuted();
```

Comment:
we don't need this unless we're actually going to record metrics for it

Reply:
Fixed in https://github.com/brave/brave-core/pull/26154/commits/b8754b2a9bd54be5fe0c2216ea7684b5bd523bad